variants/STM32F1xx/F103R(C-D-E)T/PeripheralPins_STORM32_V1_31_RC.c

```diff
@@ -0,0 +1,300 @@
+
```

Comment:
Missing Header comments
```suggestion
/*
 *******************************************************************************
 * Copyright (c) 2020-2021, STMicroelectronics
 * All rights reserved.
 *
 * This software component is licensed by ST under BSD 3-Clause license,
 * the "License"; You may not use this file except in compliance with the
 * License. You may obtain a copy of the License at:
 *                        opensource.org/licenses/BSD-3-Clause
 *
 *******************************************************************************
 */
/*
 * Automatically generated from STM32F103R(C-D-E)Tx.xml
 * CubeMX DB release 6.0.21
 */
```